.env.example

```
@@ -0,0 +1,8 @@
+# Airtable Configuration (Optional - app works with test data if not provided)
+AIRTABLE_API_KEY=your_airtable_api_key_here
+AIRTABLE_BASE_ID=your_airtable_base_id_here
+
+# Server Configuration
+NODE_ENV=development  # or 'production'
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)